crates/sncast/src/helpers/configuration.rs

```diff
@@ -71,3 +111,72 @@ impl Config for CastConfig {
         Ok(serde_json::from_value::<CastConfig>(config)?)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_networks_config_get() {
+        let networks = NetworksConfig {
+            mainnet: Some("https://mainnet.example.com".to_string()),
+            sepolia: Some("https://sepolia.example.com".to_string()),
+            devnet: Some("https://devnet.example.com".to_string()),
+        };
+
+        assert_eq!(
+            networks.get("mainnet"),
+            Some(&"https://mainnet.example.com".to_string())
+        );
+        assert_eq!(
+            networks.get("sepolia"),
+            Some(&"https://sepolia.example.com".to_string())
+        );
+        assert_eq!(
+            networks.get("devnet"),
+            Some(&"https://devnet.example.com".to_string())
+        );
+        assert_eq!(networks.get("custom"), None);
+    }
+
+    #[test]
+    fn test_networks_config_merge() {
+        let mut global = NetworksConfig {
+            mainnet: Some("https://global-mainnet.example.com".to_string()),
+            sepolia: Some("https://global-sepolia.example.com".to_string()),
+            devnet: None,
+        };
+        let local = NetworksConfig {
+            mainnet: Some("https://local-mainnet.example.com".to_string()),
+            sepolia: None,
+            devnet: None,
+        };
+
+        global.merge(&local);
```

Comment:
Having a `merge` method I would assume that both 

```rust
local.merge(global)
```

and

```rust
global.merge(local)
```

will give the same result, while currently they don't.

We have 2 possible options here:
1. Add a static method `merged` accepting `local` and `global` (this assumes local will always override global)
e.g.
```rust
NetworksConfig::merged(local, global)
```

2. Add `override_with` method, which will always override fields of given instance with values from `other` (assuming they're not `None`).
e.g.
```rust
let mut global = ...;
let local = ...;`

global.override_with(local);
```

Wdyt?